pset-functions.js

```diff
@@ -175,6 +290,17 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+
+isPalindrome = (word) => {
+  //turn to all lowercase
+  word = word.toLocaleLowerCase();
+  // test for palindrome
+  return Array.from(word).toString() === Array.from(word).reverse().toString()
```

Comment:
@anniecdavis27 nice use of `Array.from()` to turn this word into an array of letters. Alternatively, `split()` could have also been used to serve the same purpose. Take a look at the approach [used in the solution found here](https://github.com/jsd20191008/hw-01-functions-solution/blob/master/pset-functions.js#L246)